.github/workflows/basic-tests.yml

```
@@ -394,3 +394,57 @@ jobs:
         run: |
           breeze release-management generate-issue-content-core \
             --limit-pr-count 2 --previous-release 3.0.1 --current-release 3.0.2 --verbose
+
+  test-airflow-standalone:
+    timeout-minutes: 30
+    name: "Test Airflow standalone commands"
+    runs-on: ${{ fromJSON(inputs.runners) }}
+    env:
+      UV_VERSION: ${{inputs.uv-version}}
+      AIRFLOW_HOME: ~/airflow
```

```suggestion
      AIRFLOW_HOME: ~/airflow
      FORCE_COLOR: 1
```

Together with https://github.com/apache/airflow/pull/55490 it will make output in CI coloured.

---

Cool that is a nice improvement